test/manipulation.test.js

```diff
@@ -57,6 +57,20 @@ describe('manipulation', function () {
       });
     });
 
+    it('should create instance (promise variant)', function (done) {
+      Person.create({name: 'Anatoliy'})
+      .then (function (p) {
```

Comment:
@bajtos 
(I know, I know...enough about the whitespace...)

I do agree it's subjective, which is why I was pointing out my own reasoning...

FWIW, neither Google or the [Node style guide](https://github.com/felixge/node-style-guide) explicitly says *not* to put a space after `function` ...although I admit,  they don't use a space when examples contain an anonymous function...

[Douglas Crockford](http://javascript.crockford.com/code.html#function) explicitly recommends the space after function: "If the space is omited, then it can appear that the function's name is function, which is an incorrect reading."

....and not to beat a dead horse, but it looks like you are not blindly following Google's style guide.  Google and Node disagree on chaining style, and your style recommendation matches neither one (it's closer to Node's though):

[Node](https://github.com/felixge/node-style-guide#method-chaining):
```
Person
  .create(...)
  .then(...)
  .catch(...);
```
 [Google](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting):
```
Person.create(...).
    then(...).
    catch(...);
```
*Eww...* ;)

Anyways, it's your codebase...style it how you like :)

Personally, I think your chaining style is fine for js (mine works better for coffeescript...less parentheses required), but I think the space after `function` should be allowed, or even recommended...  A quick search in loopback-datasource-juggler shows that you are already doing it that way quite a bit more than without the space throughout the code, jscs or not...

(OK, I'm done now...)